index.js

```diff
@@ -324,16 +338,32 @@ morgan.token('response-time', function getResponseTimeToken(req, res, digits) {
  * current date
  */
 
+function formatTimezone(date, timezone) {
+  // Takes a date object and formats it as:
+  // Thu Oct 19 2017 12:35:19 GMT+0530 (IST)
+  const tzDate = moment(date).tz(timezone)
+}
+
 morgan.token('date', function getDateToken(req, res, format) {
+  format = format.split(',');
+  var dateFormat = format[0];
+  var timezone = format[1];
+
   var date = new Date();
 
-  switch (format) {
-    case 'clf':
-      return clfdate(date);
+  switch (dateFormat) {
+    case: 'utc':
+      return date.toUTCString();
     case 'iso':
       return date.toISOString();
```

Comment:
@TODO, include UTC offset in ISO date string so that this format can be timezone-dependent too